it/it-tests/src/test/java/it/settings/SettingsTest.java

```diff
@@ -68,14 +68,14 @@ public void should_get_settings_default_value() {
   @Test
   public void global_property_change_extension_point() throws IOException {
     orchestrator.getServer().adminWsClient().post("api/properties/create?id=globalPropertyChange.received&value=NEWVALUE");
-    assertThat(FileUtils.readFileToString(orchestrator.getServer().getLogs()).contains("Received change: NEWVALUE"));
+    assertThat(FileUtils.readFileToString(orchestrator.getServer().getLogs())).contains("Received change: [key=globalPropertyChange.received, newValue=NEWVALUE]");
```

Comment:
I see your point but IMO you want to know if the Settings notifier was changed, not if the exact log format. But as I said, it's a remark